explorer/src/providers/stats/solanaPerformanceInfo.tsx

```diff
@@ -1,6 +1,11 @@
 import { PerfSample } from "@solana/web3.js";
 import { ClusterStatsStatus } from "./solanaClusterStats";
 
+// TODO Performance stats now contain both total transaction count as well as a
+// non-vote transaction count (`numNonVoteTransaction`).  It is probably
+// interesting to show both.  The change boils down to adding a new set of
+// `avgTps`, `historyMaxTps`, `perfHistory` for non-vote transactions, or
+// something like that.
```

Comment:
We can drop the TODO comment; we already have a GH issue to update the explorer, and it is known that the update to explorer can be done once this change lands / propagates to RPC nodes on the respective clusters.
https://github.com/solana-labs/solana/issues/28174#issuecomment-1352553380

Reply:
Done.